From 4688da9929412499f7157ac957c7818afcd59764 Mon Sep 17 00:00:00 2001
From: lozohcum <lozohcum@gmail.com>
Date: Tue, 11 Feb 2014 11:01:33 +0100
Subject: [PATCH] Updater: Fix installing on older devices

---
 updater/install.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/updater/install.c b/updater/install.c
index 1140049..da87e0c 100644
--- a/updater/install.c
+++ b/updater/install.c
@@ -725,7 +725,7 @@ static struct perm_parsed_args ParsePermArgs(int argc, char** args) {
             int64_t capabilities;
             if (sscanf(args[i+1], "%" SCNi64, &capabilities) == 1) {
                 parsed.capabilities = capabilities;
-                parsed.has_capabilities = true;
+                parsed.has_capabilities = false;
             } else {
                 printf("ParsePermArgs: invalid capabilities \"%s\"\n", args[i + 1]);
                 bad++;
-- 
1.7.5.4

